### ISSUE TITLE
google colab,py310_cu118_pyt210,download fail

### ISSUE THREAD

**Issue by ajiansoft** [facebookresearch/pytorch3d#1667]
hello! my version_str=py310_cu118_pyt210,when i run it in cloab:
!pip install --no-index --no-cache-dir pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/{version_str}/download.html
it's fail:
Looking in links: https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py310_cu118_pyt210/download.html
ERROR: Could not find a version that satisfies the requirement pytorch3d (from versions: none)
ERROR: No matching distribution found for pytorch3d
but it worked well before,why?

**Comment by ajiansoft** [comment]
oh,i see,the torch version changed!

**Comment by pokeshun96** [comment]
Hi Ajian,

I hope you're doing well. I came across your comment and noticed that you faced a similar issue that I'm currently experiencing with.

Since it seems you were able to resolve this issue, I was wondering if you could provide some guidance or share how you managed to fix it?

I'd really appreciate any insights or advice you can offer.

Thank you in advance for your help!

**Comment by xeTaiz** [comment]
In the meanwhile you can install the old version of PyTorch3D on top of PyTorch 2.1.0:
```
version_str = 'py310_cu118_pyt201'
!pip install fvcore iopath
!pip install --no-index --no-cache-dir pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/{version_str}/download.html
```

@pokeshun96 There may be some issues in doing this, but so far I did not run into any ;)

**Comment by pokeshun96** [comment]
@xeTaiz I really appreciate your guidance on this issue. This solution works for me!

**Comment by bottler** [comment]
We have just released a new version 0.7.5 and it has a build matching colab, so this should now work.

**Comment by kidpaul94** [comment]
At the moment, !pip install pytorch3d does not work and !pip install git+https://github.com/facebookresearch/pytorch3d.git takes forever to be completed.
